battleship.py

```diff
@@ -172,11 +246,16 @@ def print_exit():  # Exits the game
 # Main Loop
 grid_size = 0
 
-print_welcome()
+game_mode = print_welcome()
 
 while grid_size == 0:
     grid_size = retrieve_size()
 
+if grid_size >= 32:
+    double = 4
```

Comment:
How does this "double" variable work exactly. I see it being used a lot and I am guessing it has something to do with ship count or size? We need to either name it to something more descriptive or create some comments to explain. 

Pro Tip: If you need to mass rename somehting there is a find and replace button in pycharm in the edit tab.